Integrations/src/main/java/io/deephaven/integrations/learn/Scatterer.java

```diff
@@ -0,0 +1,57 @@
+package io.deephaven.integrations.learn;
+
+import io.deephaven.datastructures.util.CollectionUtil;
+import io.deephaven.integrations.python.PythonFunction;
+import org.jpy.PyObject;
+
+import java.util.ArrayList;
+
+/**
+ * Applies scatter functions to the result of Future.get() and feeds the scattered data back into a table.
+ */
+public class Scatterer {
+
+    private final Output[] outputs;
+
+    /**
+     * Creates a new Scatterer.
+     *
+     * @param outputs array of Outputs that determine how data will be scattered back to the table.
+     */
+    public Scatterer(Output ... outputs) { this.outputs = outputs; }
+
+    /**
+     * Applies the scatter function to a subset of result.
+     *
+     * @param result        result of the call to Future.get() containing data to be scattered.
```

Comment:
Why are we talking about `Future.get()`?  That is a use case.  This class could potentially be called in other ways.

Something as simple as `Python result to be scattered into the table` may be sufficient.